src/component/playground/Playground/PlaygroundResultsTable/FeatureStatusCell/FeatureStatusCell.tsx

```diff
@@ -46,34 +17,12 @@ const StyledChipWrapper = styled(Box)(() => ({
 }));
 
 export const FeatureStatusCell = ({ enabled }: IFeatureStatusCellProps) => {
-    const theme = useTheme();
-    const icon = (
-        <ConditionallyRender
-            condition={enabled}
-            show={
-                <FeatureEnabledIcon
-                    color={theme.palette.success.main}
-                    strokeWidth="0.25"
-                />
-            }
-            elseShow={
-                <FeatureDisabledIcon
-                    color={theme.palette.error.main}
-                    strokeWidth="0.25"
-                />
-            }
-        />
-    );
-
-    const label = enabled ? 'True' : 'False';
-
     return (
         <StyledCellBox>
             <StyledChipWrapper data-loading>
-                <ConditionallyRender
-                    condition={enabled}
-                    show={<StyledTrueChip icon={icon} label={label} />}
-                    elseShow={<StyledFalseChip icon={icon} label={label} />}
+                <PlaygroundResultChip
+                    enabled={enabled}
+                    label={enabled ? 'True' : 'False'}
```

Comment:
Do we want this to show 'Unknown' if the feature is enabled in the environment but strategies are inconclusive? E.g. `feature.isEnabledInCurrentEnvironment && feature.strategies.result === 'unknown' ? "Unknown" : enabled ? 'True' : 'False'`?

Reply:
Nicolae's latest designs seem to stick with True/False for the feature and Unevaluated for the strategies (when appropriate)

Reply:
This is for the Playground Table.  Maybe we do but I am hesitant to add it now, because the other 2 have icons associated with them.

Reply:
Right. He's off today, right? In that case, we could probably add it in later. But I do think it'd be good (as mentioned in the daily) to show whether a feature is _maybe enabled_ in some way. Let's discuss with Nicolae when he's back.